src/OpenTelemetry.Api/Trace/ActivityExtensions.cs

```diff
@@ -75,6 +84,31 @@ public static Status GetStatus(this Activity activity)
             return status;
         }
 
+        /// <summary>
+        /// Gets the value of a specific tag on an <see cref="Activity"/>.
+        /// </summary>
+        /// <param name="activity">Activity instance.</param>
+        /// <param name="tagName">Case-sensitive tag name to retrieve.</param>
+        /// <returns>Tag value or null if a match was not found.</returns>
+        [MethodImpl(MethodImplOptions.AggressiveInlining)]
+        public static object GetTagValue(this Activity activity, string tagName)
```

Comment:
Curious - I wonder if there are many scenarios where people would try to get multiple tag values. If yes, a further improvement might be giving a GetTagValues overload (which takes an array/list of names and a ref to Span<object> or something allocated on the stack).

Reply:
There's actually a method a few lines up doing two lookups! 1 for status code, 1 for status description. I'll take a stab at this later. If the PR gets merged I'll do as a follow-up.